### DISCUSSION TITLE
How do I get the 3 dots back on the website bar ?

### DISCUSSION THREAD

**Discussion by Rev-0** [yokoffing/Betterfox#210] [General]
FYI I'm talking about the 3 dots that appear when opening a website that says to share link or send tab to device.

**Comment by yokoffing** [comment]
Apparently, [Mozilla removed it](https://www.reddit.com/r/firefox/comments/nwg1ls/page_action_menu_3_dots_button_missing_in_ff_890/?rdt=41216) around v. 89.

You can still right-click on a tab to share a tab.

**Reply by Rev-0** [reply]
Unfortunately, I still don't know how to use a site in app mode, do you know how to do that ?

edit: i found a solution, thx though
